unison-src/transcripts/pattern-match-coverage.md

```diff
@@ -465,7 +465,7 @@ result : '{e, Give T} r -> {e} r
 result f = handle !f with cases
        { x } -> x
        { give _ -> resume } -> result resume
-       { give A -> resume } -> result resume
+       { give T.A -> resume } -> result resume
```

Comment:
This test wasn't quite testing the right thing: since the `A` constructor is ambiguous in the namespace, it ended up as a variable, essentially testing:

```
{ give _ -> resume } -> result resume
{ give _ -> resume } -> result resume
```

rather than the intended

```
{ give _ -> resume } -> result resume
{ give T.A -> resume } -> result resume
```

but both produce the same output (complaining that the last pattern is redundant)